app/bin/tools/remove_package.dart

```diff
@@ -223,3 +244,35 @@ Future removePackageVersion(String packageName, String version) async {
   print('Version "$version" of "$packageName" got successfully removed.');
   print('NOTICE: Redis caches referencing the package will expire given time.');
 }
+
+Future _commit({
+  @required String action,
+  TransactionWrapper tx,
+  List<Model> inserts,
+  List<Key> deletes,
+}) async {
+  if ((inserts == null || inserts.isEmpty) &&
+      (deletes == null || deletes.isEmpty)) {
+    return;
+  }
+  print('');
+  print('**** About to commit the following action: $action');
+  inserts?.forEach((m) {
+    print('- insert/update: ${_debugKey(m.key)}');
+  });
+  deletes?.forEach((k) {
+    print('- delete: ${_debugKey(k)}');
+  });
+  print('Use CTRL+C to abort in 5 seconds...');
```

Comment:
We have 7 calls to `_deleteWithQuery` so the script will now take `35s` just for waiting.

I'm actually okay trusting that it does the right thing. And not showing anything for each commit.. it was just the initial confirm I thought important :)